### ISSUE TITLE
Can't compile app with Sentry 6.4.0

### ISSUE THREAD

**Issue by samphilipd** [getsentry/sentry-elixir#297]
### Description

`mix compile` yields this error:

```
Unchecked dependencies for environment dev:
* uuid (Hex package)
  could not find an app file at "_build/dev/lib/uuid/ebin/uuid.app". This may happen if the dependency was not yet compiled, or you specified the wrong application name in your deps, or the dependency indeed has no app file (then you can pass app: false as option)
** (Mix) Can't continue due to errors on dependencies
```

**Comment by ysemeniuk** [comment]
@mitchellhenke Could you push minor patch with this fix, please? Cause we need 6.4.0 version because of Plug peer fix, but we can't compile it

**Comment by mitchellhenke** [comment]
@ysemeniuk you bet, I've released `6.4.1` 🙂
